validator.js

```diff
@@ -4617,7 +4617,8 @@ function isStrongPassword(str) {
 
 var vatMatchers = {
   GB: /^GB((\d{3} \d{4} ([0-8][0-9]|9[0-6]))|(\d{9} \d{3})|(((GD[0-4])|(HA[5-9]))[0-9]{2}))$/,
-  IT: /^(IT)?[0-9]{11}$/
+  IT: /^(IT)?[0-9]{11}$/,
+  DE: /^(DE)?[0-9]{9}$/
```

Comment:
`validator.js` file is auto-generated. Please make your changes inside `src/lib/isVAT.js` file.